lib/card/query/ref_spec.rb

```diff
@@ -16,9 +21,15 @@ def initialize key, val, parent
     end
 
     def to_sql *args
-      dir, type = REFERENCE_DEFINITIONS[ @key.to_sym ]
+      dir, *type = REFERENCE_DEFINITIONS[ @key.to_sym ]
       field1, field2 = REFERENCE_FIELDS[ dir ]
-      cond = type ? ["ref_type='#{type}'"] : []
+      cond = case type.size 
+        when 0 then [] 
+        when 1 then ["ref_type='#{type.first}'"] 
```

Comment:
This would be type(.each_character).map{|letter|"ref_type='#{type.first}'"} when type.length < 2 and then the else case.  Not that case statements are bad, but aren't two cases better than three?